charts/timesketch/values.yaml

```diff
@@ -309,7 +309,9 @@ opensearch:
     ## requests:
     ##    cpu: 500m
     ##    memory: 1Gi
-    requests: {}
+    requests:
+      cpu: 250m
```

Comment:
If the opensearch params are set to 512M would it not make sense to set the pod to the same?

Reply:
Agree I bumped this to 512Mi